cosmos/x/evm/keeper/genesis.go

```diff
@@ -31,14 +31,16 @@ import (
 // InitGenesis is called during the InitGenesis.
 func (k *Keeper) InitGenesis(ctx sdk.Context, genState *core.Genesis) error {
 	// Initialize all the plugins.
+
 	for _, plugin := range k.host.GetAllPlugins() {
 		// checks whether plugin implements methods of HasGenesis and executes them if it does
 		if plugin, ok := utils.GetAs[plugins.HasGenesis](plugin); ok {
 			plugin.InitGenesis(ctx, genState)
 		}
 	}
 
-	return k.polaris.StartServices()
+	// return k.polaris.StartServices()
```

Comment:
yeah @calbera we gotta find a better spot for this. keeper isn't the move i don't think.

Reply:
future PR

Reply:
it'll cause his tests to fail

Reply:
actually @archbear you might be able to pass a mock NetworkingStack but tbh don't waste time on that